packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -34,7 +32,7 @@ let canvas: HTMLCanvasElement | null = null;
 
 function drawNative(nodeToData: Map<HostInstance, Data>, agent: Agent) {
   const nodesToDraw = [];
-  iterateNodes(nodeToData, (_, color, node) => {
+  iterateNodes(nodeToData, ({color, node}) => {
     nodesToDraw.push({node, color});
   });
```

Comment:
For React Native, we should probably add new fields here as well, including grouping via `groupAndSortNodes`.

For backwards compatibility, lets add a new event for it, something like `drawGroupedTraceUpdatesWithNames`. 

```
...

agent.emit('drawTraceUpdates', nodesToDraw);
agent.emit('drawGroupedTraceUpdatesWithNames', mergedNodes);
```